ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationEditPage.java

```diff
@@ -8,18 +8,37 @@
 import com.codeborne.selenide.SelenideElement;
 
 import io.syndesis.qe.pages.SyndesisPageObject;
+import lombok.extern.slf4j.Slf4j;
 
+@Slf4j
 public class IntegrationEditPage extends SyndesisPageObject {
-	
+
 	private static final class Element {
 		public static final By ROOT = By.cssSelector("syndesis-integrations-edit-page");
 	}
 
+	public void checkPageIsPresent(String alias) {
```

Comment:
@dsimansk,  @Stefan365  
Martin made (#181) class for validating pages based on SyndesiPageObject validate() method. I think its nice way how to standardize page presence validation. WDYT?